src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/OsmPbfProcessor.java

```diff
@@ -121,14 +119,6 @@ public void initialize(final Map<String, Object> metaData)
             logger.trace("Start loading atlas with polygon or bound {}", this.multiPolygon);
             logger.trace("With country code {}",
                     String.join(",", this.loadingOption.getCountryCodes()));
-
-            if (this.multiPolygon != null && this.loadingOption.getCountryBoundaryMap() != null)
-            {
-                logger.trace("Building grid index...");
```

Comment:
Does that mean that we would not support any country boundary map object that has not been pre-processed with grid index?

Reply:
Nope. Once this is merged, I'll send a PR for atlas-generator to handle boundary map files without grid index and boundary map files with older version of grid index.

Reply:
See https://github.com/osmlab/atlas-generator/pull/15

Reply:
So that means any local test with a boundary file manually created that does not contain the grid index would not work anymore? I agree the changes in atlas-generator make sense, but I would still like to have the option to test locally with a boundary file without grid index and have the grid index created on the fly when the slice operation starts. Is that possible? And maybe with a big warning saying the grid index is being re-created?

Reply:
I'd like to have @MikeGost's thoughts on this too... Maybe I am not seeing the whole picture here...

Reply:
I went through tests and made sure tests need grid index either work with boundary map file with grid index or generate grid index with `initializeGridIndex` call. You can now retrieve grid index by `getGridIndex` method.

Reply:
Understood, thanks!

Reply:
I think this addresses all use cases - running with a boundary file with and without grid index serialization is supported.